.github/workflows/tests.yml

```diff
@@ -65,6 +65,9 @@ jobs:
             if [[ "$x" == "net" ]]; then
               continue
             fi
+            if [[ "$x" == "worker-macros" ]]; then
+              continue
+            fi
```

Comment:
There is a CI section for workers below. We should also add the prime example to CI tests.